traitsui/testing/tester/wx/helpers.py

```diff
@@ -135,6 +227,43 @@ def key_click_text_ctrl(control, interaction, delay):
         control.WriteText(interaction.key)
 
 
+def key_click_combobox(control, interaction, delay):
+    """ Performs simulated typing of a key on the given wxComboBox
+    after a delay.
+
+    Parameters
+    ----------
+    control : wxComboBox
+        The wx Object to be acted on.
+    interaction : instance of command.KeyClick
+        The interaction object holding the key input
+        to be simulated being typed
+    delay : int
+        Time delay (in ms) in which the key click will be performed.
+    """
+    if not control.IsEditable():
+        raise Disabled("{!r} is disabled.".format(control))
+    if not control.HasFocus():
+        control.SetFocus()
+        control.SetInsertionPointEnd()
+    if interaction.key == "Enter":
+        wx.MilliSleep(delay)
+        event = wx.CommandEvent(wx.EVT_TEXT_ENTER.typeId, control.GetId())
+        control.ProcessEvent(event)
+    elif interaction.key == "Backspace":
+        wx.MilliSleep(delay)
+        start_sel, end_sel = control.GetTextSelection()
+        if start_sel != end_sel:
+            control.Remove(start_sel, end_sel)
```

Comment:
I made the change to `key_click_text_entry` and updated everywhere `key_click_text_ctrl` was used.  Unfortunately, this Pr's was branch was created before #1190 was merged.  Thus, this PR will now break things.  Should I undo the name change for now to avoid breaking anything and then update the name later in a separate PR after this is merged?

Reply:
I think in this case we can first merge master into this branch, resolve the merge conflicts, then we should see that that all we need is to rename `key_click_text_ctrl` to  `key_click_text_entry` in here:
https://github.com/enthought/traitsui/blob/1b2a3933c5313f1d4d2084f197bdb6fc8e4c1ecf/traitsui/testing/tester/wx/registry_helper.py#L38
Given the scope is so small, it seem to me a perfectly reasonable thing to simply do the renaming here.